docs/modules/ROOT/pages/migration.adoc

```diff
@@ -479,6 +479,13 @@ class WebSocketSecurityConfig: WebSocketMessageBrokerConfigurer {
 ----
 ====
 
+[[servlet-opt-in-sha256-rememberme]]
+==== Use SHA-256 in Remember Me
+
+The `TokenBasedRememberMeServices` implementation now supports SHA-256 for the Remember Me token and this is the default in Spring Security 6.
+You can start using it in 5.8 by setting the `encodingAlgorithm` property to SHA-256 while still decoding tokens that are using MD5.
```

Comment:
It may be helpful to demonstrate how an application will manage existing remember me tokens and what the migration steps for getting to `SHA256` for both matching and encoding will look like. Is there a way for us to guide them through that process?